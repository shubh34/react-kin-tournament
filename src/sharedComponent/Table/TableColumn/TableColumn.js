
import './TableColumn.scss';
import React from 'react';
import PropTypes from 'prop-types';

const TableColumn = ({ children }) => (
	<td>
		<span className="c-table-column">{children}</span>
	</td>
);


TableColumn.propTypes = {
	children: PropTypes.node.isRequired,
};

export default TableColumn;
