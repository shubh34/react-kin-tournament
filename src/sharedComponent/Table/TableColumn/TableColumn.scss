.c-table-column {
    padding: 16px;
}