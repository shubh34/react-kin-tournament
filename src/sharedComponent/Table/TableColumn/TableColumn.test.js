import React from 'react';
import { shallow } from 'enzyme';
import TableColumn from './TableColumn';

describe('TableColumn', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<TableColumn> TableColumn </TableColumn>);
		expect(wrapper).toMatchSnapshot();
	});
});
