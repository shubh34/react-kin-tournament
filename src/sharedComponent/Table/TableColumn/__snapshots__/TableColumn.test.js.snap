// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableColumn renders without crashing 1`] = `
<td>
  <span
    className="c-table-column"
  >
     TableColumn 
  </span>
</td>
`;
